setup.py

```diff
@@ -70,7 +70,6 @@
                       'corner>=2.0.1',
                       #'scikit-learn>=0.17.0',  # travis does not like scikit-learn
                       ]
```

Comment:
do not remove from the setup.py. It *is* required afterall. Even if you are installing from outside pycbc, we should not remove the dependency.